Dockerfile

```diff
@@ -82,6 +87,14 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install Criu
+ENV CRIU_VERSION 1.7
```

Comment:
@boucher do you have any ideas how to get the dump log out ot ci? If no, maybe it's worth tuning the code just to print the log on the screen in case of failure?